WordPress/Classes/Services/PostService.m

```diff
@@ -145,12 +145,13 @@ - (void)loadMorePostsOfType:(NSString *)postType
     if ([remote isKindOfClass:[PostServiceRemoteREST class]]) {
         NSString *entityName = [postType isEqualToString:PostServiceTypePage] ? NSStringFromClass([Page class]) : NSStringFromClass([Post class]);
         NSFetchRequest *request = [NSFetchRequest fetchRequestWithEntityName:entityName];
-        request.predicate = [NSPredicate predicateWithFormat:@"date_created_gmt != NULL"];
+        request.predicate = [NSPredicate predicateWithFormat:@"date_created_gmt != NULL AND blog=%@", blog];
         NSSortDescriptor *sortDescriptor = [NSSortDescriptor sortDescriptorWithKey:@"date_created_gmt" ascending:YES];
         request.sortDescriptors = @[sortDescriptor];
         Post *oldestPost = [[self.managedObjectContext executeFetchRequest:request error:nil] firstObject];
         if (oldestPost.date_created_gmt) {
             options[@"before"] = [oldestPost.date_created_gmt WordPressComJSONString];
+            options[@"order"] = @"desc";
```

Comment:
If we're setting `order`, I'd also explicitly set `order_by`, just  to be safe (and more readable)